### DISCUSSION TITLE
Where to find yhat_lower and yhat_upper?

### DISCUSSION THREAD

**Discussion by Alex-ardashev** [ourownstory/neural_prophet#270] [Q&A - get help using NeuralProphet] [answered]
Dear community,

Can you please help me to figure out how can I have the lowest expected value (yhat_lower) and highest expected value(yhat_upper) within my forecast dataset?

**Comment by ourownstory** [comment, marked as answer]
Hi @NrctcV you will find them in in the forecasted df once #255 is merged. :)

**Reply by Alex-ardashev** [reply]
@ourownstory Woah, amazing news!:) Thank you a lot!!

**Reply by whiletruelearn** [reply]
Hi , any update on this feature yet . ?
